server/manager/src/main/java/org/apache/accumulo/manager/tableOps/compact/CompactionDriver.java

```diff
@@ -211,6 +215,14 @@ public int updateAndCheckTablets(Manager manager, long tid)
                 && tabletMetadata.getSelectedFiles().getMetadataValue()
                     .equals(selectedFiles.getMetadataValue()));
 
+            if (minSelected == null || tablet.getExtent().compareTo(minSelected) < 0) {
+              minSelected = tablet.getExtent();
+            }
+
+            if (maxSelected == null || tablet.getExtent().compareTo(maxSelected) > 0) {
+              maxSelected = tablet.getExtent();
+            }
```

Comment:
This will create a range (min to max) of tablets for compaction? What if they are not entirely contiguous? Would it be better to create many events for smaller ranges?

Reply:
I thought about creating a list of tablets, but then would need to worry about buffering too much in memory.  So was not sure what to do and decided to play it safe.